src/lib/shim.native.js

```diff
@@ -1,5 +1,7 @@
 import { setupURLPolyfill } from 'react-native-url-polyfill'
 import { XMLHttpRequest as XHR2 } from 'xhr2-cookies'
+import SQLite from 'react-native-sqlite-2'
```

Comment:
why this had to be moved here ?
doesn't it breaking anything else (e.g. prod build / unit tests) ?

Reply:
It is used for shimming indexDB only, unit tests passed successfully

Reply:
@roma-hladilka ^